x-pack/plugins/transform/public/app/sections/transform_management/components/action_delete/delete_action_modal.tsx

```diff
@@ -86,6 +103,19 @@ export const DeleteActionModal: FC<DeleteAction> = ({
   const deleteModalContent = (
     <>
       <EuiFlexGroup direction="column" gutterSize="none">
+        {hasManagedTransforms ? (
+          <p>
+            <>
+              <EuiText>
+                <FormattedMessage
+                  id="xpack.transform.transformList.deleteManagedTransformDescription"
+                  defaultMessage="This transform was deployed as part of a module; deleting it might impact other parts of the product."
```

Comment:
Transforms are not part of modules, so we should definitely drop the term 'module' here, and say something like `This transforms was provided by Elastic...`

Reply:
Updated here https://github.com/elastic/kibana/pull/122305/commits/427be5b2ff13c2453e9b03611547f978baa19cbc